mage_ai/streaming/sources/kafka.py

```diff
@@ -42,13 +42,18 @@ class KafkaConfig(BaseConfig):
     api_version: str = '0.10.2'
     batch_size: int = DEFAULT_BATCH_SIZE
     timeout_ms: int = DEFAULT_TIMEOUT_MS
+    auto_offset_reset: str = 'latest'
```

Comment:
where do you use this field?

Reply:
This field is used in [consumer_kwargs](https://github.com/mage-ai/mage-ai/pull/4226/files#diff-d183d9cbac5329d37f58e32ec12a23789222c7be7978f22dca04286bb353e6daR88)  to initialize a kafka consumer